OasisDEX/oasis-borrow
# added atp flag

# [UI | Auto-Take Profit Banner](https://app.shortcut.com/oazo-apps/story/5512/ui-auto-take-profit-banner)

  
## Changes 👷‍♀️
- Added ATP Banner 
![image](https://user-images.githubusercontent.com/6871923/190381112-ef4984cb-a2bc-4700-bf72-38f1d1cd5c45.png)

- Added dropdown
![image](https://user-images.githubusercontent.com/6871923/190381016-01226b6c-cb2f-490d-ada0-99e9790cf6ec.png)

  
## How to test 🧪
  
- Check if banner displays correctly as in figma design
- Check if auto take profit is displayed in dropdown, if states of form change correctly
